src/app/pages/OnDemandRadioPage/index.jsx

```diff
@@ -63,7 +63,11 @@ const StyledMessageContainer = styled.div`
   margin-bottom: ${GEL_SPACING_QUAD};
 `;
 
+// iframe padding set to keep scrub bar and duration in view
 const StyledAudioPlayer = styled(AVPlayer)`
+  amp-iframe {
+    overflow: visible !important;
```

Comment:
```suggestion
    width: calc(100% + ${GEL_SPACING_DBL});
    @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
      width: calc(100% + ${GEL_SPACING_QUAD});
```

Reply:
That should resolve the above @RichardPK

Reply:
Awesome looks like that did it